examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
OK so it sortof works but there's still weird stuff that happens. For example:

```python
tmin = -1.
tmax = tmin + 1023. / raw.info['sfreq']
epochs = mne.Epochs(raw, events, event_id, tmin, tmax)
print(epochs.get_data().shape)
```
will print shape (..., 1024)

but if I do
```python
tmin, tmax = -1., 1.
raw.info['bads'] += ['MEG 2443']  # bads
epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                    proj=True, baseline=(None, 0), preload=True,
                    reject=dict(grad=4000e-13, eog=150e-6))
tmax = tmin + 1023. / raw.info['sfreq']
epochs.crop(tmin, tmax)
print(epochs.get_data().shape)
```

Now it prints shape (..., 1023)

So depending on whether you define the tmin/tmax before turning into epochs, vs cropping the epochs after the fact, then you are off by 1. So I think I ran into this inconsistency and just said I would index manually because I wanted to be sure that it was correct.